### DISCUSSION TITLE
[2기 - B] 김희아

### DISCUSSION THREAD

**Discussion by Gangneng** [super-corini/programmers_kdt_II#362] [2기 Monthly Project I]
* 웹 사이트 : http://yellowgangneng.pythonanywhere.com/
* git : https://github.com/Gangneng/eda

# 해결 중 어려웠던 점

어떤 데이터가 어떤 느낌의 연관을 갖고 있을거란 느낌도 오고, 가설은 설정할 수 있었으나 그를 잘 표현해 줄 그래프를 생각해내는 것이 어려웠습니다. 아직 데이터 분석 자체에 감을 못잡은 느낌이 없지 않아 있어요.
그리고 저주받은 미적감각으로 정말 보기 안 좋은 사이트를 만든 것 같아... 부끄럽습니다....

극복은.. 앞으로 해내야 겠죠...? 틈틈히 캐글 대회에도 참가해보면서 데이터 분석 마스터가 되는 그 날까지 힘내봐야겠습니다. AI의 완성도는 데이터에 달려있기도 하니까 더더욱 데이터에 대해 공부를 열심히 해야겠어요.

# 새로 공부한 것

배포하는 법을 새로 공부했습니다. 비록 pythonanywhere라는 굉장히 편리하게 웹사이트 배포를 도와주는 사이트의 도움을 받았지만요.. 다음엔 꼭 aws를 이용해 배포할 수 있도록 해야겠습니다. 1달동안 감사했습니다. 남은 기간도 열심히 해보겠습니다!

**Comment by super-corini** [comment]
- 데이터에 답이 있습니다. 이 데이터의 분포를 파악해보시고 (scatter plot, bar plot 등) 이 데이터의 타입 등을 바탕으로 연관된 시각화를 진행해보시면 좋을 것 같아요.
- pythonanywhere를 사용하는 것도, AWS를 사용하는 것도 좋습니다. 다만 그것을 해냈다는 것이 더 중요하죠 😎 1달동안의 희아님의 노력이 이후의 개발인생에 있어서 나침반이 되었으면 합니다. 😸
